package.json

```diff
@@ -187,6 +187,7 @@
 		"irc-upd": "^0.10.0",
 		"material-design-icons": "^3.0.1",
 		"material-icons": "^0.3.1",
+		"mustache": "^4.0.1",
```

Comment:
This is already included in devDependencies, please delete it from here.

Reply:
Done. Mustache removed from dependencies.